buf/registry/descriptor/v1/file_descriptor_proto_service.proto

```diff
@@ -0,0 +1,130 @@
+  // We include this information in a separate message as FileDescriptorProtos are not
+  // extendable, and we do not want to modify the FileDescriptorProto shape.
+  bool exclude_file_descriptor_proto_extensions = 5;
+  // Only include specific symbols (and symbols that they reference, if exclude_imports is not set).
+  //
+  // A symbol is a package, message, enum, service, method, or extension.
+  //
+  // Symbols are referenced by their fully-qualified name. For example, if "Foo" is a message within
+  // the package "foo.v1", the fully-qualified name is "foo.v1.Foo". Fully-qualfied names should
+  // not start with a period, that is specify "foo.v1.Foo", not ".foo.v1.Foo".
+  //
+  // If no symbols are provided, all symbols are returned.
+  repeated string symbols = 6;
+  // An explicit, opt-in field mask for the fields populated on the returned FileDescriptorProtos.
+  //
+  // For example, if you would only like to return the names of messages, use the mask:
+  //
+  //   {
+  //     paths: [
+  //       "message_type.name",
+  //       "message_type.nested_type.name"
+  //     ]
+  //   }
+  //
+  // The filters exclude_imports, exclude_source_code_info, exclude_source_retention_options and
+  // symbols continue to be respected even if this field is set. Specifically, if exclude_source_code_info
+  // is set, but this mask includes source_code_info, SourceCodeInfo will continue to be excluded.
+  google.protobuf.FieldMask field_mask = 7;
+}
+
+message GetFileDescriptorProtosResponse {
+  option (buf.validate.message).cel = {
+    id: "len_file_descriptor_protos_equals_len_extensions",
+    message: "the length of file_descriptor_protos must equal the length of file_descriptor_proto_extensions if file_descriptor_proto_extensions is present",
+    expression: "this.file_descriptor_proto_extensions.size() == 0 || this.file_descriptor_protos.size() == this.file_descriptor_proto_extensions.size()"
+  };
+
+  // The FileDescriptorProtos representing the referenced Module, Label, or Commit.
+  repeated google.protobuf.FileDescriptorProto file_descriptor_protos = 1 [(buf.validate.field).required = true];
+  // Additional information about each FileDescriptorProto.
+  //
+  // We include this information in a separate message as FileDescriptorProtos are not
+  // extendable, and we do not want to modify the FileDescriptorProto shape.
+  //
+  // If present, the length of this field will be equal to the length of the file_descriptor_protos field, and each
+  // value corresponds to its corresponding value at the same index within file_descriptor_protos.
+  repeated FileDescriptorProtoExtension file_descriptor_proto_extensions = 2;
+  // The version of the compiler used to create the FileDescriptorProtos.
+  //
+  // This will match the semantics used within CodeGeneratorRequests.
+  google.protobuf.compiler.Version compiler_version = 3 [(buf.validate.field).required = true];
```

Comment:
Does this imply we're doing something new here? I thought we used to always set the version to nil/unset since version of the Buf CLI weren't really comparable to versions of `protoc`.